activerecord/lib/active_record/core.rb

```diff
@@ -24,6 +24,14 @@ module Core
       # their relevant queries. Defaults to false.
       mattr_accessor :verbose_query_logs, instance_writer: false, default: false
 
+      ##
+      # :singletom-method:
+      #
+      # Specifies the BacktraceCleaner used to identiy the relevent caller for the
+      # verbose_query_logs.
+
+      mattr_accessor :backtrace_cleaner, instance_writer: false, default: BacktraceCleaner.new
```

Comment:
I think we can store the cleaner in the only place that uses it, in the log subscriber. What do you think?

Reply:
@rafaelfranca The `backtrace_cleaner` has been moved to be attached to the `LogSubscriber` as per your suggestion.